UnitTests/CommonTestUtils/NoAssertContext.cs

```diff
@@ -0,0 +1,130 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Concurrent;
+using System.Diagnostics;
+using System.Threading;
+
+namespace System
+{
+    /// <summary>
+    ///  Use (within a using) to eat asserts.
```

Comment:
```suggestion
    ///  Use (within a using) to eat asserts, e.g. Debug.Fail.
```

Reply:
You're welcome to send a PR to https://github.com/dotnet/winforms/pull/2191 😉